src/Butil/Bit.Butil/Scripts/userAgent.ts

```
@@ -0,0 +1,595 @@
+var BitButil = BitButil || {};
```

Biome

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)